lib/Varien/Data/Form/Element/Multiselect.php

```diff
@@ -57,8 +57,10 @@ public function getElementHtml()
     {
         $this->addClass('select multiselect');
         $html = '';
-        if ($this->getCanBeEmpty() && empty($this->_data['disabled'])) {
-            $html .= '<input type="hidden" name="' . parent::getName() . '" value="" />';
+        if ($this->getCanBeEmpty()) {
+            $html .= '<input type="hidden" name="' . parent::getName() . '" value=""';
+            $html .= empty($this->_data['disabled']) ? '' : ' disabled="disabled"';
```

Comment:
because this hidden disabled field is getting enabled by when unchecking the "Use Default Value"

and then when no Items are selected in the Multiselect, the hidden Input is sent via POST.
For why, see this: https://stackoverflow.com/questions/1132303/how-to-post-empty-select-multiple-html-elements-if-empty

Then for Magento, it gets an empty value for that Form Element and sets the data to empty array.

otherwise the field would be missing from the POST, and then Magento wouldn't touch the data.
Or falls back on using the Data from the Default Value